modules/README.md

```diff
@@ -116,8 +117,10 @@ Kudos to @mfw78
 
 There are continuous efforts to expand the types of signatures supported by the EVM beyond the currently predominant secp256k1 elliptic curve. For example, a signature scheme gaining popularity is based on the secp256r1 elliptic curve (see EIP-7212). Signature Validators allow accounts to support new standards and enable use-cases such as Passkeys-enabled smart accounts, BLS/Schnorr or quantum-secure signatures.
 
+When a contract wants to verify account signature, it should call `isValidSignature(bytes32,bytes)` on the account and the account calls the `SignatureValidatorManager` for validation.
+
 References:
-- https://github.com/zerodevapp/kernel/blob/main/src/validator/IValidator.sol
+- https://github.com/zerodevapp/kernel/blob/main/src/interfaces/IValidator.sol
```

Comment:
Should we move this to the end of the signature validator section? This was originally added as a something to get inspiration from. Also might be worth adding a reference to 6900.